File: example/optimizely/php/sample-OPT.php

FeatureFlag::start("your_env_id", "your_api_key");
$visitor = FeatureFlag::newVisitor("your_visitor_id")->build();
$visitor->updateContext("isVip", true);
$visitor->fetchFlags();

$flag = $visitor->getFeatureVariableDouble('my_feature_key', 'OPT-flag-php', 'user_123', $attributes);

File: example/optimizely/java/sample-OPT.java
Visitor visitor = featureFlag.newVisitor("visitor_unique_id").build();
visitor.updateContext("isVip", true);
visitor.fetchFlags().whenComplete((instance, error) -> {
    
    Flag<String> btnColorFlag = visitor.getFeatureVariableString("my_feature_key", "OPT-flag-java", "user_123", attributes);

});

